libs/remix-simulator/src/methods/accounts.ts

```diff
@@ -85,6 +86,10 @@ export class Web3Accounts {
     }
   }
 
+  eth_requestAccounts (_payload, cb) {
```

Comment:
I think these changes are already in master...

Reply:
No this was part of your PR for ethers 6, it's not on master

Reply:
https://github.com/ethereum/remix-project/pull/5188/files#diff-d91b9465dbf7e11a3da4f3c022a8649e7e509c5ccca17641898910875277e982R89